docs/src/adoc/index.adoc

```diff
@@ -684,6 +684,10 @@ handle.createQuery("select * from emotion where emoticon = <sticker>;")
     .list();
 ----
 
+Bindings and definitions are usually separate.  You can link them in a limited manner
+using the `stmt.defineNamedBindings()` or `@DefineNamedBindings` customizers, which will
+define a boolean indicating whether the value is present and not a `NullArgument` (bind of a `null`).
```

Comment:
I think users who know nothing about this feature won't be informed clearly enough by this paragraph. `define a boolean` without a `for each` of some sorts implies a single boolean, and `the` in `the value` refers to something unspecified.

Maybe something like this?

>Bindings and definitions are usually independent. You can use the `stmt.defineNamedBindings()` or `@DefineNamedBindings` customizers to make e.g. conditionals easier for `TemplateEngines` to process: a definition/attribute will be added for each binding, with the same name (unless something is already defined with said name), and with the value being a boolean that indicates if the bound object is a `NullArgument` (`false`) or not (`true`).

Reply:
I used some of your wording and included a concrete example